spec/fixture_app/config/application.rb

```diff
@@ -1,5 +1,6 @@
 module Fixture
   class Application < Rory::Application
     turn_off_request_logging!
+    filter_parameters :orcas, :noodles
```

Comment:
Ok, its good to hear your thoughts on this. I am not saying what I suggested was best, just different and was curious to hear your thinking around the approach you took.